src/scss/components/_button.scss

```diff
@@ -222,4 +203,26 @@ $button-padding-h: $grid-margin-2;
       color: var(--of-button-danger-bg);
     }
   }
+
+  // anchor variants, these map to utrecht components:
+  // * UtrechtButton (the default)
+  // * UtrechtButtonLink (`a` component)
+  // * UtrechtLinkButton (`button` component)
+  @include bem.modifier('anchor') {
+    @include button-theme(
```

Comment:
currently the best way to handle all the accessibility stuff, we'll revisit later